packages/fingerprint-generator/src/fingerprint-generator.ts

```diff
@@ -24,9 +24,34 @@ export type ScreenFingerprint = {
     hasHDR: boolean;
 }
 
+export type Brand = {
+    brand: string;
+    version: string;
+}
+
+export type UserAgentData = {
+    brands: Brand[];
+    mobile: boolean;
+    platform: string;
+    architecture: string;
+    bitness: string;
+    fullVersionList: Brand[];
+    model: string;
+    platformVersion: string;
+    uaFullVersion: string;
+}
+
+export interface ExtraProperties {
```

Comment:
generally speaking, we prefer interfaces over types for objects, but since all the other object types here are types, lets stay consistent and change only this one to be a `type`